core/src/main/kotlin/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractorV2.kt

```diff
@@ -60,7 +60,7 @@ fun runScheduleMetadataExtractor(
     // Compute stops
     val stops = ArrayList<ResultStops>()
     for (stop in legacyStops) {
-        val stopTime = envelopeWithStops.interpolateTotalTime(stop.position) - stop.duration
+        val stopTime = envelopeWithStops.interpolateDepartureFrom(stop.position) - stop.duration
```

Comment:
```suggestion
        val stopTime = envelopeWithStops.interpolateArrivalAt(stop.position)
```

Same changes as the v1

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7609/commits/b7f044ebd08664f02948f7aafb19982fee469f65